.gitignore

```diff
@@ -33,3 +33,7 @@ docs/build/
 .DS_Store
 .AppleDouble
 .LSOverride
+
+# Visual Studio configuration and output files
+out
+/.vs
```

Comment:
I don't think this should go into the PIConGPU dev, since PIConGPU does not especially favour VS

Reply:
If this is really a big problem for you, then I can remove it. However, you ignore so many other tools' temporary files (like MacOS stuff, VS Code, even Code::Blocks and Netbeans), I don't think those additional ones would hurt.